ansible_collections/arista/avd/docs/release-notes/5.x.x.md

```diff
@@ -132,6 +132,12 @@ Previously it also accepted a string, which was autoconverted to a list.
 
 See the [porting guide](../porting-guides/5.x.x.md#) for details.
 
+#### Changes in MTU values and support for `p2p_uplinks_mtu` in platform settings
+
+Starting AVD 5.0.0, `p2p_uplinks_mtu` support added in platform settings and default MTU for Dps1 and WAN HA interfaces changed to 9194.
```

Comment:
Looks like it is two changes in one which may be hard for people to find I guess we could go with two titles? 
@ClausHolbechArista

Reply:
the platform setting is a feature, and should not be highlighted. The change is the new default MTU for WAN routers. How we get there (or rather revert it) is for the porting guide to show.

Reply:
updated